wind_poses:
  # gripper: 0.6
  p1: [-0.042982405068913244, -0.08693575852277718, 0.12311220541596413, -0.7050142492024855, 0.7091930826634916, 3.817123400988356e-05, 0.0002801190220939662]
  p2: [-0.04690423039527489, -0.08842879068798726, 0.024821821600198746, -0.7050561817224293, 0.7091513987797472, 4.906779059662099e-05, 0.0002679852026444051]
  # gripper: 0.95
  # drag
  p3: [-0.11978151611417226, -0.11749990928109466, 0.024832438677549362, -0.705196437622675, 0.7090117309171873, 0.0005807822330284536, -0.00011170541914312348]
  # gripper: 0.35
  p4: [-0.06337161461275931, -0.09694196805507874, -0.026835322380065932, -0.6687121385018177, 0.7075085233478547, -0.1561318820921635, 0.16696886118322374]
  # p4: [-0.07932879416705624, -0.09953548665353545, -0.026840440928936005, 0.7047642254809383, -0.6716086749687117, 0.1646255209916894, -0.15857998621276073]
  # gripper: 0.965
  # wind
  p5: [-0.08472505528732929, 0.02132845095811109, 0.0011081211268901825, -0.7052915016277728, 0.7089171327828475, 0.0006183718200911816, -0.00011914227023191832]
  p6: [-0.0599701001650311, 0.047486586674669096, 0.02156522497534752, -0.7052513462457981, 0.7089570925458389, 0.0006071940708575405, -0.00010422475958688671]
  p7: [-0.0599701001650311, 0.047486586674669096, 0.02156522497534752, -0.7052513462457981, 0.7089570925458389, 0.0006071940708575405, -0.00010422475958688671]
  # p6: [-0.08481992452202608, 0.04774299292476547, -0.014008082449436188, -0.7052259937689604, 0.7089823479950763, 0.0005553070617678941, -0.00013992154211028753]
  # p7: [-0.01761728004283965, 0.05902574994036902, 0.044460006058216095, -0.7051383780069764, 0.7090695227586331, 0.0005088870150906453, -0.00014418925276306232]

  p8: [-0.011775693514257841, 0.04781384245167175, 0.10474899783730507, -0.7052208836788139, 0.7089874348482215, 0.000549388243710416, -0.00014360769504665342]
  p9: [-0.0163762323860102, 0.04017857730284757, 0.10715610161423683, -0.7051883148431357, 0.7090198650038322, 0.0005045345973204483, -0.00013070381167449817]
  # p10: [-0.01593559901160544, -0.07075695441019086, 0.11794846132397652, -0.7051828464413472, 0.7090253136302814, 0.0005011087608179796, -8.12795811573235e-05]
  p10: [-0.05627571179869799, -0.10783634737122105, 0.11783538386225702, -0.7051853489553762, 0.7090228230746374, 0.0005018563556616178, -9.010079896770574e-05]

  p11: [-0.09545772151093723, -0.16001215720067996, -0.010144665837287903, -0.7051717494727153, 0.7090363558355585, 0.0004944172731074555, -7.348864839220394e-05]
  p12: [-0.06984640002077971, -0.23425814219416943, -0.013613421469926834, -0.7051541766323236, 0.709053833683718, 0.000488738209394298, -9.61840685812264e-05]
  p13: [-0.062465500242867206, -0.24300011336293334, 0.03565685823559761, -0.7051519099782104, 0.7090560873564773, 0.0004885324971540824, -0.00010086032199300324]
  p14: [-0.028202340615085808, -0.2130819146130076, 0.07566609606146812, -0.7051476604137205, 0.7090603133599978, 0.0004875783847766991, -0.00010629297803067837]
  p15: [-0.0285627830031584, -0.16887821510485404, 0.09640344604849815, -0.7051483217024263, 0.7090596550391939, 0.0004890912282184553, -0.00010385432945401146]
  p16: [-0.08127178652972031, -0.04588428419706195, 0.0963968001306057, -0.7051477553165543, 0.7090602176464892, 0.0004902016104244017, -0.00010308016069108214]
  
  # p11: [-0.015691876219364187, -0.1604157331166786  2, 0.10211144760251045, -0.7052172952651354, 0.7089909534353862, 0.0006102063105640875, -0.00014850578167328952]
  # p12: [-0.03823091810801532, -0.23459092441580004, 0.07936236634850502, -0.7051899069910506, 0.7090181736140407, 0.0006238769817651373, -0.00018799062620495052]
  # p13: [-0.03829418927565918, -0.23456678130226033, 0.05288127437233925, -0.705199714466739, 0.709008329866793, 0.0007068066858184145, -0.00022653882568391962]
  # p14: [-0.07358440989106751, -0.22570025303710783, 0.0510520376265049, -0.7051942447836648, 0.7090138936951565, 0.0005682861685913908, -0.00022961285034350334]
  # p15: [-0.07227849344792409, -0.22461215980211924, 0.01600724831223488, -0.7051881814117129, 0.7090200054745269, 0.00047656672750032847, -0.0001830800292070566]
  # # p16: [-0.05529408277127401, -0.2343805514271764, 0.015934709459543228, 0.8169814011137275, -0.5603890163119615, 0.11642030841243357, -0.07036939974389952]
  # p16: [-0.08195422469830503, -0.24188026248329264, 0.014440681785345091, 0.8169625077232093, -0.5604030918218056, 0.11646275987293946, -0.0704064003762746]
  # # p17: [-0.0662998184211143, -0.23463155714192613, -0.015269394963979721, 0.8170281564403605, -0.5603248223768954, 0.11640902752061151, -0.07035640215127485]
  # p17: [-0.09322606905885931, -0.2421919361008762, -0.012774597853422151, 0.8170198654086511, -0.5603363231494439, 0.11641089135978243, -0.0703580049448007]
  # # p18: [-0.07764045061598039, -0.18955553151895732, -0.01527320221066475, 0.8170281564403605, -0.5603248223768954, 0.11640902752061151, -0.07035640215127485]
  # p18: [-0.13360334873602264, -0.16020637615951766, -0.02646806091070175, 0.8170252859831573, -0.5603294975860169, 0.11640819023182336, -0.07035388721381194]
  # p19: [-0.09082283575888224, -0.08670753289333677, -0.015260506421327591, 0.8170334257290439, -0.5603189867499037, 0.11640468476865141, -0.07034887129823018]
  

  p17: [-0.11200659796316968, -0.013312343041227692, -0.015261482447385788, -0.7053108287862561, 0.7088976380970622, 0.0008513664048065765, -0.00022061844021471468]
  p18: [-0.08350211670305105, 0.02914826610051302, -0.015258606523275375, -0.7053180938586958, 0.708890412081783, 0.0008466376784674506, -0.0002309517206719659]
  p19: [-0.08360525167861937, 0.05044612464664, -0.015250742435455322, -0.7053033717238074, 0.7089050421700978, 0.0008548310061732983, -0.00025354445733005314]
  p20: [-0.060252984650812996, 0.06302553859575205, -0.004787370562553406, -0.7053398236241153, 0.7088687420009167, 0.0008930571099376215, -0.0002056038378622715]
  p21: [-0.05131452574214726, 0.05608541476295126, 0.053622741252183914, -0.7052343710308036, 0.7089737380835466, 0.0008254557455588271, -0.00019810967126546495]
  p22: [-0.021586251513669552, 0.05681273093255962, 0.07531354203820229, -0.7051808499300313, 0.7090270427970551, 0.0007712438732002972, -0.00016327066542630653]
  p23: [-0.021426590974918322, 0.016802208324640244, 0.09842676296830177, -0.705158856493411, 0.7090489278073409, 0.000765181270895204, -0.00013993494212491658]
  p24: [-0.02123391881582813, -0.03739413548197029, 0.11235256865620613, -0.7051580067481231, 0.7090497616191787, 0.0007755279748247123, -0.00014008269215265076]
  p25: [-0.020565942719940757, -0.21958848529718195, 0.11234748736023903, -0.7051668799258136, 0.7090409161083225, 0.0007927726721756096, -0.00014916921418512956]
  p26: [-0.04473461398356665, -0.2196332068066696, 0.07132040336728096, -0.7051775419229229, 0.7090301761952783, 0.0009008502898181936, -0.00017911111652343213]
  p27: [-0.04762808354017313, -0.22596216553926712, 0.04793887212872505, -0.7052064291981157, 0.7090013818318884, 0.0009470783035838782, -0.00018926384485430938]
  p28: [-0.07282969734016533, -0.22601585540277686, -0.008526835590600967, -0.7052816392039302, 0.7089262770807554, 0.0011243979791935245, -0.0002807052226872796]
  p29: [-0.09389943795369714, -0.14765010067781353, -0.014094743877649307, -0.7052569771046152, 0.7089506412120783, 0.0012134808322295627, -0.0003347155701636958]
  # gripper: 0.35